doc/DEVELOPMENT.rst

```diff
@@ -216,3 +216,39 @@ Automatic
 
 To do the above steps automatically, run ``scripts/macos-deps-install.sh``.
 
+Graphical Assets
+----------------
+
+Taisei's GFX library is made up of a collection of sprites, shaders, and a few
+3D models. The 3D models are exclusively used for background scenery, while the
```

Comment:
Technically the 3D models are not used exclusively for backgrounds - the HUD is a flat 3D model with a cutout for the game's viewport :smile: That was done as a small rendering optimization. But technically nothing prevents you from using models in 2D context in general, they are just a bunch of packed vertex data after all. So perhaps "mostly" or "almost exclusively" is a better way of putting it.